integral_view/forms/system_date_time_form.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class date_time_form(forms.Form):
+    system_time = forms.CharField(required=False)
+    system_date = forms.CharField(required=False)
+
+    def clean(self):
+        cd = super(date_time_form, self).clean()
+        if cd["system_time"] == "" and cd["system_date"] == "":
```

Comment:
It is possible that those fields are not even there in cd as it is not a required field. So for such cases you should always say it like -
if ('system_time' not in cd or cd["system_time"] == '') and ('system_date' not in cd or cd["system_date"] == ""):

Also, please use single quotes for string constants instead of double unless absolutely required..